Infrastructure/checkService.js

```diff
@@ -0,0 +1,70 @@
+const ping = require('ping');
+const exec = require('child_process').exec;
+const dig = require('node-dig-dns');
+const {
+  servers: serverList,
+  workstations: workstationList,
+  dnsQueryDomain,
+  sshUser
+} = require('./config.js');
+
+const DIG_ARG = 'ns';
```

Comment:
```suggestion
const DIG_ARGS = ['ns'];
```

This could then be [spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) (e.g. `[dnsQueryDomain, ...DIG_ARGS]`) to support multiple args in future.

Otherwise it's probably ok to inline these arguments if they're only being used once.

Reply:
That makes sense. I've included the use of spread syntax to the args.